packages/carbon-web-components/src/components/structured-list/structured-list.ts

```diff
@@ -30,6 +30,18 @@ class BXStructuredList extends FocusMixin(LitElement) {
   @property({ attribute: 'selection-name' })
   selectionName = '';
 
+  /**
+   * Specify if structured list is condensed, default is false
+   */
+  @property({ type: Boolean, reflect: true, attribute: 'condensed' })
```

Comment:
```suggestion
  @property({ type: Boolean, reflect: true })
```
I think `attribute` may be redundant here?